connect/connect.go

```diff
@@ -3,24 +3,22 @@ package connect
 import (
 	"fmt"
 
+	"github.com/joshmedeski/sesh/config"
 	"github.com/joshmedeski/sesh/session"
 	"github.com/joshmedeski/sesh/tmux"
 	"github.com/joshmedeski/sesh/zoxide"
 )
 
-func Connect(choice string, alwaysSwitch bool, command string) error {
-	session, err := session.Determine(choice)
+func Connect(choice string, alwaysSwitch bool, command string, config *config.Config) error {
+	session, err := session.Determine(choice, config)
 	if err != nil {
 		return fmt.Errorf("unable to connect to %q: %w", choice, err)
 	}
-
-	if err := zoxide.Add(session.Path); err != nil {
+	if err = zoxide.Add(session.Path); err != nil {
```

Comment:
```suggestion
	if err := zoxide.Add(session.Path); err != nil {
```

We need a `:` here please